packages/next/build/webpack-config.ts

```diff
@@ -282,7 +282,7 @@ export default async function getBaseWebpackConfig(
           reuseExistingChunk: true,
         },
       },
-      maxInitialRequests: 20,
+      maxInitialRequests: 15,
```

Comment:
I thought we wanted to increase this instead of decreasing it. If we're still experimenting with it can we remove it from this PR and ship CSS improvements first?